### PR TITLE
Translated help page @Codepages.NoAutoDetectCP into English.

### PR DESCRIPTION
**Pull request by MKadaner**
Also, MSVC 15.8.1 compile errors.

### CODE REVIEW

File: far/FarEng.hlf.m4 [hunk 1]
```diff
@@ -5372,25 +5370,29 @@ If current value of an option is other than the default, the option is marked wi
...
-    Если параметр равен строке #"-1"#, то в зависимости от того включён или нет показ
-только избранных кодовых страниц (#Ctrl+H# в ~меню кодовых страниц~@CodePagesMenu@),
-будут разрешены только стандартные (ANSI, OEM, Unicode) и избранные, либо все.
-
-    В противном случае параметр должен быть списком запрещённых номеров кодовых страниц.
-Например "1250,1252,1253,1255,855,10005,28592,28595,28597,28598,38598".
-
-    Юникодные кодовые страницы (1200, 1201, 65001) проверяются отдельно от UCD, поэтому не могут быть
-отключены, даже если они есть в списке исключения.
+    This string parameter defines the code pages, which will be excluded
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: This is a restrictive clause, no comma is required.

File: far/FarEng.hlf.m4 [hunk 2]
```diff
@@ -5372,25 +5370,29 @@ If current value of an option is other than the default, the option is marked wi
...
+from Universal Codepage Detector (UCD) autodetect. Sometimes, especially
+on small files, UCD annoyingly choses wrong code pages.
+
+    The default value is empty string #""#. In this case all code pages
+detectable by UCD (about 20, much less than can be installed on the
+computer) are enabled.
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: Maybe drop `on the computer`?

File: far/FarRus.hlf.m4 [hunk 3]
```diff
@@ -5513,25 +5512,30 @@ $ #Редактор конфигурации#
 
 @Codepages.NoAutoDetectCP
 $ #far:config Codepages.NoAutoDetectCP#
-    Это строка, при помощи которой можно задать кодовые страницы, которые будут исключены
-из автоопределения кодовой страницы UCD (Universal Codepage Detector). Иногда (особенно на
-небольших файлах) UCD назойливо выбирает неподходящие кодовые страницы.
+    Этот строковый параметр задаёт кодовые страницы, которые будут
```
Comment (alabuzhev) [inline far/FarRus.hlf.m4]: `кодовые страницы, которые будут исключены из автоматического определения кодовой страницы` sounds tautological, maybe drop the last two words?

File: far/changelog_eng [hunk 4]
```diff
@@ -3,19 +3,21 @@ This file is a translation of the main russian changelog and is provided by volu
...
 =======================================================================================
 
-drkns 09.08.2018 21:06:04 +0100 - build 5255
+MZK 18.08.2018 21:41:08 -0700 - build 5257
 
-1. SQLite does not use rand_s() since 3.21 so Win2k support which has been introduced in 4710 now can be removed.
+1. Translated help page @Codepages.NoAutoDetectCP into English.
 
-2. A couple of issues.
+2. MSVC 15.8.1 compile errors.
```
Comment (alabuzhev) [inline far/changelog_eng]: Sorry, I've added this already.

File: far/config.hpp [hunk 5]
```diff
@@ -893,7 +893,7 @@ class Options: noncopyable
 	// заменена на умолчательную ANSI или OEM, в зависимости от настроек.
 	// пример: L"1250,1252,1253,1255,855,10005,28592,28595,28597,28598,38598,65001"
 	// Если строка пустая никакой фильтрации кодовых страниц в UCD детекте не будет.
-	// Если "-1", то в зависимости CPMenuMode (Ctrl-H в меню кодовых страниц фильтрация UCD либо будет
+	// Если "-1", то в зависимости CPMenuMode (Ctrl-H в меню кодовых) страниц фильтрация UCD либо будет
```
[the remark this answers is not in the document]
Comment (MKadaner) [reply far/config.hpp]: Thank you. Fixed.

File: far/FarEng.hlf.m4 (hunk 1 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Thank you. My bad.

File: far/FarEng.hlf.m4 (hunk 2 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: I'd rather keep it in sync with the Russian version. And I think `(около двух десятков, гораздо меньше, чем может быть установлено)` would be somewhat unclear.

If you think we can drop the computer in both places, I am OK with this.

File: far/FarRus.hlf.m4 (hunk 3 above)
Comment (MKadaner) [reply far/FarRus.hlf.m4]: Yeah! Thank you.

File: far/changelog_eng (hunk 4 above)
Comment (MKadaner) [reply far/changelog_eng]: Thank you.

File: far/changelog_eng [hunk 6]
```diff
@@ -32,14 +36,6 @@ drkns 03.08.2018 12:13:31 +0100 - build 5253
 
 1. Continuation of 5249.
 
-drkns 05.08.2018 14:05:25 +0100 - build 5254
```
[the remark this answers is not in the document]
Comment (MKadaner) [reply far/changelog_eng]: They are duplicated. Please double check least I mess the things up.

File: far/FarEng.hlf.m4 (hunk 2 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Good point. Like a font cartridge installed in an HP printer. Please see if you like this:
`(about 20, much less than there is usually available in the system)`
and
`(около двух десятков,гораздо меньше, чем обычно доступно в системе)`

File: far/changelog_eng (hunk 6 above)
[the remark this answers is not in the document]
Comment (alabuzhev) [reply far/changelog_eng]: Ah right, thanks.

**Comment by alabuzhev** [comment]
Merged, thanks.

**Pull request merged**